src/main/java/net/mcreator/element/types/Item.java

```diff
@@ -134,4 +167,36 @@ public boolean hasInventory() {
 		return guiBoundTo != null && !guiBoundTo.isEmpty() && !guiBoundTo.equals("<NONE>");
 	}
 
+	public static class ModelEntry {

```

Comment:
If you plan to add item states to block items too, maybe you should move this inner class to another file. This way we will avoid redundant code.

Reply:
Item model only needs one texture, block model needs up to 6 of them. Also, when block states are supported, bounding boxes will be defined separately for each block model. So it's convenient to leave this class defined for item ME only.